src/assets/styles/base/_base.css

```diff
@@ -170,7 +177,9 @@ main section:last-of-type:not(:first-of-type) {
 
 a:not([class]) {
     color: var(--fl-linkFgColor, currentColor);
-    font-weight: var(--font-weight-semibold);
+    font-size: var(--fl-enhance-font-size);
+    font-weight: var(--fl-enhance-font-weight, var(--font-weight-semibold));
+    text-decoration: var(--fl-enhance-text-decoration);
```

Comment:
This causes link underlines to not appear when enhanced-input is turned off.

Reply:
Thanks for catching this. I've pushed up a fix.